ingest_data/sagemaker-keyspaces/README.md

```diff
@@ -1,24 +1,28 @@
-# Train Machine Learning Models using Amazon Keyspaces as a Data Source  
+# Train Machine Learning Models using Amazon Keyspaces as a Data Source 
 
-In this notebook we will provide step-by-step instruction to use SageMaker to ingest customer data from Amazon Keyspaces and train a clustering model that allowed you to segment customers. You could use this information for targeted marketing, greatly improving your business KPI.
+Please read [Train machine learning models using Amazon Keyspaces as a data source](https://aws.amazon.com/blogs/machine-learning/train-machine-learning-models-using-amazon-keyspaces-as-a-data-source/) blog for more detailed instructions to run this notebook.
+
+
+We will provide step-by-step instructions to use SageMaker to ingest customer data from Amazon Keyspaces and train a clustering model that would enable you to segment customers. This information can be used for targeted marketing, greatly improving your business KPI.
```

Comment:
Similar edits to this sentence as to later in the readme - "We ~~will~~ provide", "model that ~~would enable~~ enables you"

Reply:
Edited.